### PR TITLE
PP-12509 Surface more at-a-glance information on payment details page

### PR DESCRIPTION
**Pull request by alexbishop1**
- Shuffle some things around at the top of the payment details page
- On the payment details page, put tags near the top to indicate if various features applied to the payment: recurring, delayed capture, payment link, call centre via admin tool, external phone, MOTO
- On the payment details page, use logos for the card brand (if available), show whether the card is debit or credit (if known) and include an Apple Pay or Google Pay logo if a digital wallet was used
- On the payment details page, show the first six digits of the card number (if available) in addition to the last four digits
- On the payment details page, show the language and source (payment created using API, payment link, call centre agent took payment over phone using admin tool, external phone payment taken outside of GOV.UK Pay, payment created before we recorded source) of the payment in the processing details table
- On the payments list page, add a method column with a logo for the card brand (if available), whether the card is debit or credit (if known) and an Apple Pay or Google Pay logo if a digital wallet was used
- Use blue tags for things that are in-progress to free up grey for less dynamic things
- Use an orange tag for declined payments to match other statuses that indicate the payment is terminal and was not successful
- Tweak some labels for correctness and clarity

# Top of payment details page
image

# Processing details table on payment details page
image

# Main table on payments list page
image

### CODE REVIEW

File: src/web/modules/transactions/list.njk [hunk 1]
```diff
@@ -140,11 +145,11 @@
...
         disabled
       {% elseif filters.gateway_payout_id %}
-        href="/transactions?page={{ set.page + 1 }}&account={{ accountId }}&gateway_payout_id={{ filters.gateway_payout_id }}"
+        href="/transactions?page={{ set.page + 1 }}&amp;account={{ accountId }}&amp;gateway_payout_id={{ filters.gateway_payout_id }}"
       {% elseif filters.agreement_id %}
-        href="/transactions?page={{ set.page + 1 }}&account={{ accountId }}&agreement_id={{ filters.agreement_id }}"
+        href="/transactions?page={{ set.page + 1 }}&amp;account={{ accountId }}&amp;agreement_id={{ filters.agreement_id }}"
       {% else %}
-        href="/transactions?page={{ set.page + 1 }}&status={{ selectedStatus }}{{ linkQueryParams }}"
+        href="/transactions?page={{ set.page + 1 }}&amp;status={{ selectedStatus }}{{ linkQueryParams }}"
```
Comment (katstevens) [inline src/web/modules/transactions/list.njk:152]: We don't encode these elsewhere (eg on the agreements page nav links). Is this causing a problem?

Comment (alexbishop1) [reply src/web/modules/transactions/list.njk:152]: Other than that it annoys me, no, not really. I just thought I’d fix it while I was editing that page.

Comment (katstevens) [reply src/web/modules/transactions/list.njk:152]: OK - we should probably be consistent across the app if we do this, there are a a few dotted around the templates

**Review by oswaldquek — APPROVED** [review]
Nice changes. That macro functionality is cool too.

File: src/web/modules/transactions/list.njk (hunk 1 above)
Comment (alexbishop1) [reply src/web/modules/transactions/list.njk:152]: I’ll consistently fix them when I have a consistent few minutes.

**Pull request merged**